doc/import_ngeo_config.py

```diff
@@ -21,7 +21,9 @@ def _format_type(type_definition: dict[str, Any]) -> str:
         if "typeArguments" in type_definition:
             postfix = f'\\<{", ".join([_format_type(t) for t in type_definition["typeArguments"]])}>'
 
-        return f":ref:`integrator_guide_ngeo_properties_{type_definition['name']}`{postfix}"
+        if type_definition["name"].lower() not in ():
```

Comment:
is this really ever going to do something ?

Reply:
Oups remaining :-), Thanks